kolibri/core/static/assets/fonts/noto-subset.ar.css

```diff
@@ -32,618 +32,613 @@ ABQCMgAKAkIAAAINAAACrQAeAtoAHgFXAAABDQAAAq0AHgLaAB4BlQAeAeYAHgH7AAACPgAA\
-AAEAAAAAAAEACQAAAAEAAQAAAAEABAAAAAEAAwALABgAVgDIASgBsAGiAbAByAHsAgQCSAAB\
-AAAAAQAIAAIAdAAYABIAGAAcACAAJAAoADQAOAA8AEAARABIAEwAUABUAF4AYgBmAGoAbgB0\
-ABQAgACEAAEAAAABAAgAAgA2ABgAEQAXABsAHwAjACcAMwA3ADsAPwBDAEcASwBPAFMAXQBh\
-AGUAaQBtAHMAEwB/AIMAAQAYAA8AFQAZAB0AIQAlADEANQA5AD0AQQBFAEkATQBRAFsAXwBj\
-AGcAawBxAHsAfQCBAAEAAAABAAgAAQAGAAEAAQAnAAMABQAHAAkADwAVABkAHQAhACUAKQAr\
-AC0ALwAxADUAOQA9AEEARQBJAE0AUQBbAF8AYwBnAGsAcQB1AHcAeQB7AH0AgQCMAI4AkACS\
-AAQACQABAAgAAQBoAAMADAAaAEYAAQAEAJUABABmAGUAcgAFAAwAFAAaACAAJgCUAAMAIwB8\
-AI0AAgAEAI8AAgAGAJEAAgAIAJMAAgAKAAQACgAQABYAHACMAAIABACOAAIABgCQAAIACACS\
-AAIACgABAAMAAwBlAGYAAQAAAAEACAABABQATAABAAAAAQAIAAEABgBWAAIAAQC0AL0AAAAG\
-AAAAAQAIAAMAAQASAAEAKgAAAAEAAAAIAAIAAQEAAQkAAAABAAAAAQAIAAEABv/2AAIAAQEK\
-ARMAAAAEAAAAAQAIAAEANgABAAgABQAMABQAHAAiACgA/gADAN4A4QD/AAMA3gDiAPsAAgDe\
-APwAAgDhAP0AAgDiAAEAAQDeAAEAAAABAAgAAQAGABwAAQABAKE=') format('woff');
+AAAAAAAAAAAAAAAAAAD/zgAAAAAAAAAAAAAAAAAAAAAAAAAAAAD/9gABACAAvAC+AL8AwADF\
+AMYAyQDKAMsAzgDPANAA0QDSANMA1ADWANcA2gDbAN0A4ADhAOIA4wDlAOcA6QDqAOsA7AD3\
+AAEAvAA8AAIAAAAKAAMABAAAAAAAAAAAAA8ACQAAAAAAAwASAAMAAAAAAAwABgAIAAgADwAH\
+AA4AAAABAAAAAAAAAAAAEQAAAAEAAAAAAAEAAQAAAAAAAAANAAAACwAAAAUABQAQAAUAAAAA\
+AAAAAAAAAAAAAAAAAAAAAAARAAEAvABAAAUAAAACAAAAAAAAAAIAAAAAAAAAAAAAAAAAAgAA\
+AAIAAAAAAA4ABgALAAsAAAAJABEAAAAEAAgAAQABAAEACgANAAgAAAAIAAMAAwABAAMAAQAD\
+ABAADAADAAcABwAHAAcADwAAAAAAAAAAAAAAAAAAAAAAAAAKAAoACgAKAAoAAAABAAAACgB8\
+ASAABURGTFQAIGFyYWIAOmN5cmwATGdyZWsAYGxhdG4AYAAEAAAAAP//AAgACQAKAAsADAAN\
+AA4ADwAQAAQAAAAA//8ABAABAAMABgAIAAQAAAAA//8ABQAAAAIABAAFAAcABAAAAAD//wAE\
+AAAAAgAEAAcAEWRub20AcmZpbmEAeGZyYWMAfmluaXQAhmxpZ2EAjGxvY2wAaG1lZGkAkm51\
+bXIAmHJsaWcAnmRub20AcmZpbmEAeGZyYWMAfmluaXQAhmxpZ2EAjG1lZGkAkm51bXIAmHJs\
+aWcAngAAAAMACgAKAAoAAAABAAUAAAABAAIAAAACAAYABwAAAAEAAAAAAAEACQAAAAEAAQAA\
+AAEABAAAAAEAAwALABgAVgDIASgBsAGiAbAByAHsAgQCSAABAAAAAQAIAAIAdAAYABIAGAAc\
+ACAAJAAoADQAOAA8AEAARABIAEwAUABUAF4AYgBmAGoAbgB0ABQAgACEAAEAAAABAAgAAgA2\
+ABgAEQAXABsAHwAjACcAMwA3ADsAPwBDAEcASwBPAFMAXQBhAGUAaQBtAHMAEwB/AIMAAQAY\
+AA8AFQAZAB0AIQAlADEANQA5AD0AQQBFAEkATQBRAFsAXwBjAGcAawBxAHsAfQCBAAEAAAAB\
+AAgAAQAGAAEAAQAnAAMABQAHAAkADwAVABkAHQAhACUAKQArAC0ALwAxADUAOQA9AEEARQBJ\
+AE0AUQBbAF8AYwBnAGsAcQB1AHcAeQB7AH0AgQCMAI4AkACSAAQACQABAAgAAQBoAAMADAAa\
+AEYAAQAEAJUABABmAGUAcgAFAAwAFAAaACAAJgCUAAMAIwB8AI0AAgAEAI8AAgAGAJEAAgAI\
+AJMAAgAKAAQACgAQABYAHACMAAIABACOAAIABgCQAAIACACSAAIACgABAAMAAwBlAGYAAQAA\
+AAEACAABABQASQABAAAAAQAIAAEABgBSAAIAAQCzALsAAAAGAAAAAQAIAAMAAQASAAEAKgAA\
+AAEAAAAIAAIAAQD8AQQAAAABAAAAAQAIAAEABv/3AAIAAQEFAQ0AAAAEAAAAAQAIAAEANgAB\
+AAgABQAMABQAHAAiACgA+gADANsA3gD7AAMA2wDfAPcAAgDbAPgAAgDeAPkAAgDfAAEAAQDb\
+AAEAAAABAAgAAQAGABsAAQABAKA=') format('woff');
   font-style: normal;
   font-weight: normal;
-  unicode-range: U+20,U+30-39,U+41-5A,U+5F,U+61-69,U+6C-7A,U+621-63A,U+640-64A,U+660-661,U+669;
+  unicode-range: U+20,U+30-36,U+38-39,U+41-49,U+4B-5A,U+5F,U+61-69,U+6C-7A,U+621-63A,U+640-64A,U+660-661,U+669;
```

Comment:
@rtibbles my guess is that it's the unicode range update which triggered such a big diff in the css-embedded font, but still not completely sure.

One thing that's interesting is the unicode range is fairly sensitive to the _exact_ characters used. Here for example it changes `U+30-39` to `U+30-36,U+38-39`, essentially removing the number `7` from the subset font... if we wanted to reduce churn, we could add some "smoothing" to the subset code to make it a bit less precise but more stable.

Reply:
I don't think it's particularly worrissome - although it would be nice to have a content hash of these CSS files in the filename, as we've discussed previously!